### PR TITLE
Improve switch-statements in disassembly ##disasm

### PR DESCRIPTION
**Pull request by radare**
* Fix default case flag name

before:
Screenshot 2019-10-26 at 07 14 56

after:
Screenshot 2019-10-26 at 07 11 55

### CODE REVIEW

File: libr/core/disasm.c [hunk 1]
```diff
@@ -2172,6 +2172,31 @@ static int flagCmp(const void *a, const void *b) {
 	return strcmp (fa->name, fb->name);
 }
 
+static void __preline_flag(RDisasmState *ds, RFlagItem *flag) {
+	ds_newline (ds);
+	ds_begin_line (ds);
+	ds_pre_line (ds);
```
Comment (kazarmy) [inline libr/core/disasm.c]: One day there should proper documentation as to how to begin a line and how to print comments, but right now the tests will have to do.

**Comment by radare** [comment]
Yep disasm.c needs a lot of love still

**Comment by XVilka** [comment]
New one is better, agree

**Comment by radare** [comment]
hopefully ready to merge after fixing all the tests

**Pull request merged**